pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/KafkaPartitionLevelConnectionHandler.java

```diff
@@ -60,6 +60,7 @@ public KafkaPartitionLevelConnectionHandler(String clientId, StreamConfig stream
     if (_config.getKafkaIsolationLevel() != null) {
       consumerProp.put(ConsumerConfig.ISOLATION_LEVEL_CONFIG, _config.getKafkaIsolationLevel());
     }
+    consumerProp.put(ConsumerConfig.CLIENT_ID_CONFIG, _clientId);
```

Comment:
Can this change be made for 0.9 consumer as well?

Reply:
0.9 consumer seems to be setting this up correctly. No change required. ty!